### PR TITLE
Document jslib-aws version v0.8.1

### PR DESCRIPTION
**Pull request by oleiade**
This Pull Request documents changes in the latest [jslib-aws version v0.8.1](https://github.com/grafana/k6-jslib-aws/releases/tag/v0.8.1).

Most notably it:
- bumps examples version to use `v0.8.1`.
- documents the new `S3Client.copyObject` operation.

### CODE REVIEW

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: 'S3Client.copyObject'
+description: 'S3Client.copyBucket copies an object from a bucket to another'
+excerpt: 'S3Client.copyBucket copies an object from a bucket to another'
+---
+
+`S3Client.copyBucket` copies an object from one bucket to another.
```
Comment (immavalls) [inline src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md:7]:
```suggestion
`S3Client.copyObject` copies an object from one bucket to another.
```

**Review by immavalls — APPROVED** [review]
LGTM. Minor typo, and I'm also missing the `copyObject` listed in https://mdr-ci.staging.k6.io/docs/refs/pull/1273/merge/javascript-api/jslib/aws/s3client/, shall we add it?

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+| Parameter            | Type   | Description                                        |
+| :------------------- | :----- | :------------------------------------------------- |
+| sourceBucket         | string | Name of the bucket to copy the object from.        |
+| sourceObjectKey      | string | Name of the object to copy from the source bucket. |
+| destinationBucket    | string | Name of the bucket to copy the object to.          |
+| destinationObjectKey | string | Name of the object to copy from the source bucket. |
```
Comment (heitortsergent) [inline src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md:14]: Another question, for the `destinationObjectKey` parameter, does that have to match the `sourceObjectKey` value? Or can the user just set it to the name of the new object? If it's the latter, I think we could change the description for `destinationObjectKey`.

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 S3Client.md
Comment (heitortsergent) [inline src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 S3Client.md:1]: @oleiade would you mind adding the new method to this file as well?

**Review by heitortsergent — COMMENTED** [review]
Overall it looks good, thanks for updating the versions on all the examples! I left a couple of suggestions, and a couple of questions as well.

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md (hunk 1 above)
Comment (oleiade) [reply src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md:7]: I must have been really tired indeed 🤦🏻 Thanks a lot, great catch 👀

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oleiade) [reply src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md:14]: You're absolutely right 🤝 I'll adjust it accordingly 👍🏻

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 S3Client.md
Comment (oleiade) [reply src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 S3Client.md:1]: I completely forgot indeed. Will add it 🙇🏻

File: src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md (hunk 2 above)
Comment (oleiade) [reply src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md:14]: The latter. Judging by the methods REST API endpoint documentation, there is no requirement for the source object key and the destination object key to match, I will update the descriptions accordingly 👍🏻

**Comment by oleiade** [comment]
Thanks a lot for taking the time to review this @heitortsergent 🙇🏻 Much appreciated!

**Review by heitortsergent — APPROVED** [review]
LGTM, thanks for making all the updates! :shipit:

**Pull request merged**